fe/fe-core/src/main/java/com/starrocks/load/routineload/RoutineLoadTaskScheduler.java

```diff
@@ -112,6 +113,7 @@ private void process() throws InterruptedException {
         updateBackendSlotIfNecessary();
 
         int idleSlotNum = routineLoadManager.getClusterIdleSlotNum();
+        MetricRepo.GAUGE_ROUTINE_LOAD_IDLE_SLOT_NUM.setValue((long) idleSlotNum);
```

Comment:
Why is this metric added?

Reply:
I think it's necessary to monitor idle slot num, we added it to our alarm system.

Do I need to remove this metric?